client/src/app/metadata-schema-form/metadata-form.service.ts

```diff
@@ -25,10 +25,12 @@ export class MetadataFormService {
     if (!dataCopy) {
       return dataCopy;
     }
+    console.log(dataCopy);
```

Comment:
Do we need this log to the browser?

Reply:
No. Sorry, I missed this before I merged. Hotfix done on dev branch in f40e9e6bbc45861efac16639b1cdee7a4c28738c